app/assets/stylesheets/govuk_publishing_components/components/_inverse-header.scss

```diff
@@ -27,3 +27,15 @@
 .gem-c-inverse-header--padding-top {
   padding-top: govuk-spacing(3);
 }
+
+@include govuk-media-query($media-type: print) {
+  .gem-c-inverse-header {
+    background-color: unset;
+    border-bottom: 2pt solid $govuk-print-text-colour;
```

Comment:
What's this new border for?

Reply:
@andysellick This was missing the first time around. It's to implement the separation of components as noted in the PR description: "For components that imply layout and separation with their background colours, those aspects should be communicated with borders or other solutions that work better in print."

Without this border, big inverted headers don't appear to be separated from other page content. The border is now doing the job in print that the background colour does on screen.

Reply:
Okay, thanks.